src/verilog/verilog_synthesis.cpp

```diff
@@ -1400,8 +1452,32 @@ Function: verilog_synthesist::synth_decl
 \*******************************************************************/
 
 void verilog_synthesist::synth_decl(const verilog_declt &statement) {
```

Comment:
May I ask to move the opening brace to the next line while working on this code?

Reply:
done